ingestion/Dockerfile

```diff
@@ -31,7 +32,13 @@ WORKDIR /ingestion
 COPY ingestion /ingestion
 
 ARG INGESTION_DEPENDENCY=all
-RUN pip install ".[${INGESTION_DEPENDENCY}]"
+RUN pip install ".[postgres,mysql,redshift]"
+
+# Uninstalling psycopg2-binary and installing psycopg2 instead 
+# because the psycopg2-binary generates a architecture specific error 
+# while authrenticating connection with the airflow, psycopg2 solves this error 
+RUN pip uninstall psycopg2-binary -y
+RUN pip install psycopg2
```

Comment:
As per comments above, did you mean `pip uninstall` instead ?

Reply:
cc - @pmbrull ^^

Reply:
let's see if we can use `psycopg2` directly on the setup

Reply:
after checking with @ulixius9, agree with the idea of leaving this on the Docker not to have any unnecessary installs from users on other environments.

Thanks folks